#Message the communicates a point of where to go to dodge obstacles and a flag for if there are any obstacles 
Header header
 
#Point to navigate around the obstacle
geometry_msgs/PointStamped point

